phpstan.neon

```diff
@@ -60,6 +60,9 @@ parameters:
     excludePaths:
         - src/cli/src/Flow/CLI/Command/*
         - src/core/etl/src/Flow/ETL/Formatter/ASCII/ASCIITable.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelectorAll.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelector.php
+        - src/core/etl/src/Flow/ETL/Row/Entry/HTMLEntry.php
```

Comment:
how many suppression those would require in file?
(I'm not home today, so sorry if I'm asking redundant questions, not everything is fully visible from phone 😅)

Reply:
Few are some not really fixable by comment as they are generics related that is saying given class doesn't exists 🫣

Reply:
It's called baseline which you don't like 🤣

Reply:
I know that tool and what baseline is (which is a list of such ignored errors, nothing more). Done.